src/main/java/mekanism/client/MekanismKeyHandler.java

```diff
@@ -43,20 +47,23 @@
 public class MekanismKeyHandler extends MekKeyHandler {
 
     public static final String keybindCategory = Mekanism.MOD_NAME;
-    public static KeyBinding modeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.mode"),
-          Keyboard.KEY_M, keybindCategory);
-    public static KeyBinding armorModeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.armorMode"),
-          Keyboard.KEY_G, keybindCategory);
-    public static KeyBinding voiceKey = new KeyBinding("Mekanism " + LangUtils.localize("key.voice"), Keyboard.KEY_U,
-          keybindCategory);
-    public static KeyBinding freeRunnerModeSwitchKey = new KeyBinding("Mekanism " + LangUtils.localize("key.feetMode"),
-          Keyboard.KEY_H, keybindCategory);
+    public static KeyBinding modeSwitchKey = new KeyBinding("key.mode", Keyboard.KEY_M, keybindCategory);
```

Comment:
might want to do something like "key.mekanism.mode" just because I feel like "key.mode" may not be an uncommon name for a keybinding. Given I am pretty sure it saves this stuff in the options.txt file for keybindings.

Reply:
yeah we definitely need to do that as part of solving #4698
Will check what impact that has on options.txt as part of this

Reply:
I think at the very least changing the keybind will not save previously saved keybinding settings. If I remember correctly our current way is not ideal anyways as it has a space in the path.

Reply:
Yeah, I'll write a datafixer for it